### PR TITLE
Add haproxy httpcheck

### PR DESCRIPTION
**Pull request by amrutp-redhat**
Added the required changes. Will try to rehearse this PR to see the results.

### CODE REVIEW

**Comment by amrutp-redhat** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.18-ocp-e2e-ovn-remote-libvirt-multi-z-z

File: libvirt/haproxy/haproxy_lnxocp01.cfg [hunk 1]
```diff
@@ -151,32 +151,42 @@ frontend https-all
...
-    server      master1 192.168.2.11:6443 check
-    server      master2 192.168.2.12:6443 check
-    server      master3 192.168.2.13:6443 check
+    server      bootstrap 192.168.2.10:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master1 192.168.2.11:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master2 192.168.2.12:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master3 192.168.2.13:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
 backend masters-03
     mode        tcp
+
```
Comment (prb112) [inline libvirt/haproxy/haproxy_lnxocp01.cfg]: missing the `option httpchk GET /readyz HTTP/1.0`

Comment (amrutp-redhat) [reply libvirt/haproxy/haproxy_lnxocp01.cfg]: Done. Thank you @prb112

**Review by telpelt — COMMENTED** [review]
Please note bootstrap configuration differs from [documentation](https://docs.openshift.com/container-platform/4.17/installing/installing_ibm_z/installing-ibm-z-kvm.html#installation-load-balancing-user-infra-example_installing-ibm-z-kvm).

**Comment by prb112** [comment]
Hey @telpelt I think both ways are correct with bootstrap. 'bootstrap' won't be ready in that case.

**Comment by amrutp-redhat** [comment]
changed bootstrap config as per document.

**Comment by prb112** [comment]
Hey @amrutp-redhat perhaps remove `[WIP]:` from the title. Thanks, Paul

**Review by prb112 — APPROVED** [review]

**Review by telpelt — APPROVED** [review]

**Comment by amrutp-redhat** [comment]
@tvardema i can't see merge pull request button. please help.

**Pull request merged**